tensorboard/webapp/notification_center/_views/notification_center_test.ts

```diff
@@ -28,7 +28,6 @@ import * as selectors from '../_redux/notification_center_selectors';
 import {CategoryEnum} from '../_redux/notification_center_types';
 import {NotificationCenterComponent} from './notification_center_component';
 import {NotificationCenterContainer} from './notification_center_container';
-const testAction = createAction('[Notification] Notification Bell Clicked');
```

Comment:
This is simple removed as cleanup?

Reply:
Yes, just a cleanup of an unused target.